pkg/apis/mysql/v1alpha1/mysqlcluster_types.go

```diff
@@ -130,6 +130,12 @@ type MysqlClusterSpec struct {
 	// +optional
 	ServerIDOffset *int `json:"serverIDOffset,omitempty"`
 
+	// CompressCommand is a command to use for compressing the backup.
+	CompressCommand []string `json:"compressCommand,omitempty"`
+
+	// DecompressCommand is a command to use for decompressing the backup.
+	DecompressCommand []string `json:"decompressCommand,omitempty"`
```

Comment:
Let's use `Backup` as a prefix.

```suggestion
	// BackupCompressCommand is a command to use for compressing the backup.
	BackupCompressCommand []string `json:"compressCommand,omitempty"`

	// BackupDecompressCommand is a command to use for decompressing the backup.
	BackupDecompressCommand []string `json:"decompressCommand,omitempty"`
```

Reply:
Done: https://github.com/presslabs/mysql-operator/pull/624/commits/7322f53ddc8236bd21d67a0d2228fd6eece5a196